source/reference/about-mongosync.txt

```diff
@@ -0,0 +1,157 @@
+.. _about-mongosync:
+
+====================
+About ``mongosync``
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+The ``mongosync`` binary is the primary process used in {+c2c-product-name+}. 
+``mongosync`` migrates data from a source cluster to a destination cluster and 
+keeps the clusters in continuous sync. You can use ``mongosync`` to create 
+dedicated analytics, development, or testing clusters that mirror your 
+production environment. 
+
+``mongosync`` keeps track of its current actions through 
+:ref:`states <c2c-states>`. ``mongosync`` enters different states depending on 
+the requests it receives. The current ``mongosync`` state determines which API 
+operations you can run. 
+
+.. note:: 
+
+   Most state transitions happen as the result of an API call. The exceptions 
+   are:
+
+   - The transition from ``COMMITTING`` to ``COMMITTED``
+   - The transition from ``REVERSING`` to ``RUNNING``
+
+To view the current state of ``mongosync``, use the :ref:`/progress
+<c2c-api-progress>` endpoint.
+
+``mongosync`` Diagram
+-----------------------
+
+The following diagram illustrates the relationship between ``mongosync`` states 
+and API operations:
+
+.. figure:: /images/mongosync.drawio.bakedsvg.svg
+   :alt: Diagram of relationship between ``mongosync`` states and API operations
+   :figwidth: 600px
+
+How ``mongosync`` Works
```

Comment:
[suggestion] Based on our [concept page prototype](https://www.mongodb.com/docs/meta/style-guide/information-types/prototypes/concept/#details), we might want to title this section as "Details" since it helps unpack each process in mongosync.

Reply:
Like this idea, fixing now